frontend/src/userlist.jsx

```
@@ -0,0 +1,42 @@
+    fetchUsers()
+  }
+
+  return (
+    <div>
+      <h1>Users</h1>
+      <button onClick={() => fetchUsers()}>Load Users</button>
+      {loading && <p>Loading...</p>}
+      {error && <p>Error fetching users</p>}
+      <ul>
+        {users.map((user) => {
+          return <li>{user.name} - {user.email}</li>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)